pages/Wishlist/wishlist.html

```diff
@@ -0,0 +1,159 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>FreshStore</title>
+
+	<!-- font awesome cdn link  -->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+
+	<!-- custom css file cdn link  -->
+	<link rel="stylesheet" href="/style.css">
+	<link rel="stylesheet" href="wishlist.css">
+
+</head>
+
+<body>
+
+	<!-- header section starts  -->
+
+	<header class="header">
+
+		<a href="#" class="logo"> <i class="fab fa-canadian-maple-leaf"></i> FreshStore </a>
+
+		<form action="" class="search-form">
+			<input type="search" name="" placeholder="Search" id="searchBox">
+			<label for="searchBox" class="fas fa-search"></label>
+		</form>
+
+		<div class="icons">
+			<div class="fas fa-search" id="search-btn"></div>
+			<div class="fas fa-moon" id="theme-btn"></div>
+			<div class="fas fa-shopping-cart"></div>
+			<div class="fas fa-bars" id="menu-btn"></div>
+		</div>
+
+		<nav class="navbar">
+			<a href="/index.html" class="active-page">Home</a>
+			<a href="./pages/Product Page/product.html">Products</a>
+			<a href="/pages/Cart Page/cart.html">Cart</a>
+			<a href="/pages/Wishlist/wishlist.html">Wishlist</a>
+			<a href="/pages/Signup-login/login.html">Login</a>
+			<a href="/pages/Signup-login/signup.html">Signin</a>
+		</nav>
+
+	</header>
+
+	<!-- header section ends -->
+
+
+
+	<!-- Cards section starts  -->
+
+	<section class="cards" id="cards">
+
+		<h1 class="heading">Wishlist</h1>
+
+		<div class="box-container">
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-4.png" alt="">
+
+				</div>
+				<div class="content">
+					<h2 class="card-title">Title</h2>
+					<div class="price"> 290.99 <span>350.99</span> </div>
+					<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero, vitae.</p>
+					<div class="btn-container">
+						<a href="#" class="btn"> Buy Now!</a>
+						<a href="#" class="btn-secondary"> Remove!</a>
+					</div>
+				</div>
+			</div>
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-1.png" alt="">
+				</div>
+				<div class="content">
+					<h2 class="card-title">Title</h2>
+					<div class="price"> 290.99 <span>350.99</span> </div>
+					<p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Vero, vitae.</p>
+					<div class="btn-container">
+						<a href="#" class="btn"> Buy Now!</a>
+						<a href="#" class="btn-secondary"> Remove!</a>
+					</div>
+				</div>
+			</div>
+
+			<div class="box">
+				<div class="image">
+					<img src="/images/product-2.png" alt="">
+				</div>
+				<div class="content">
+					<h2 class="card-title">Title</h2>
+					<div class="price"> 290.99 <span>350.99</span> </div>
```

Comment:
You can use &#x20b9; for rupee symbol if you like...
<div class="price">&#x20b9; 290.99 <span>350.99</span> </div>
 Rest everything looks good.